src/dependencies/token/IBadERC20.sol

```diff
@@ -0,0 +1,7 @@
+pragma solidity 0.6.4;
+
+interface IBadERC20 {
```

Comment:
Can we name this something like `FlexibleIERC20` and add a description in Natspec to the top about what it does? I just don't like having the `Bad~` namespacing with our prod contracts lol

Reply:
The `BadERC20Token.sol` naming is fine, because that accurately describes what it is and it's only in tests, but our interface can handle both that and normal ERC20 tokens.